invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -355,7 +355,7 @@ export class RDMDepositForm extends Component {
                   <DatesField
                     fieldPath="metadata.dates"
                     options={this.vocabularies.metadata.dates}
-                    showEmptyValue
+                    showEmptyValue={false}
```

Comment:
this is actually not fixing the regression, because it is hiding the field, if I understood correctly from Javi's comment